staging/src/k8s.io/cli-runtime/pkg/resource/helper_test.go

```diff
@@ -120,7 +120,7 @@ func TestHelperDelete(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			client := &fake.RESTClient{
-				NegotiatedSerializer: scheme.Codecs,
+				NegotiatedSerializer: scheme.Codecs.WithoutConversion(),
```

Comment:
Did you need to change that?

Reply:
Yes – there was a test that failed that this change fixes. I unresolved the discussion above – see there. In the thread, I mentioned I actually don't understand how this fixes the broken test.